.changeset/rotten-lemons-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@nordeck/matrix-neoboard-widget': major
+'@nordeck/matrix-neoboard-react-sdk': major
+---
+
+Migrate neoboard to vitejs + vitest
```

Comment:
also explain a bit more why it is a major bump, e.g. ES module, possible required migrations due to tooling